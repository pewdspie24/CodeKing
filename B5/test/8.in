10
4 14
6 10
7 16
10 13
9 10
8 11
4 10
5 13
10 10
10 13
