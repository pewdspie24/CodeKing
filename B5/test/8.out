65536
8589934592
22876792454961
4194304
6103515625
1977326743
1000000000000
9765625
13060694016
379749833583241
14348907
1000000000000
505447028499293771
6103515625
3486784401
131072
100000000000
4294967296
232630513987207
205891132094649
68719476736
43046721
16677181699666569
387420489
131072
4782969
100000000000
2541865828329
22876792454961
34522712143931
19683
16677181699666569
470184984576
1000000000000000
1220703125
2357947691
177147
1594323
43046721
177147
129140163
68719476736
1000000000000
2541865828329
4194304
16677181699666569
67108864
10000000000000000
512
10000000000
4294967296
4177248169415651
268435456
134217728
6103515625
13841287201
13841287201
17179869184
35184372088832
35184372088832
129140163
177147
67108864
549755813888
1048576
48828125
8192
10000000000000000
